docs/references/subcommands/report.md

```diff
@@ -24,17 +24,19 @@ fossa report attribution --timeout 60
 
 Where `60` is the maximum number of seconds to wait for the report to be downloaded.
 
-### Print report as JSON
+### Specifying a report format
 
-By default, `fossa report` displays issues in a human-readable format. To instead print issues as JSON, use:
+`fossa report` supports customizing the format used to render a report via the `--format` flag.
+Available options are:
+- `json`
+- `markdown`
+- `spdx`
 
-```sh
-fossa report attribution --json
-```
+For example, to render the report in JSON format, use `fossa report attribution --format json`.
+
+In earlier versions of the FOSSA CLI, the `--json` flag was used to denote rendering the report in JSON format.
+This flag is still supported and takes precedence over the `--format` flag if present, but is considered deprecated.
```

Comment:
Minor nit: "considered" weakens this statement.  We are deprecating it, right?  We should just say that.